server/Dockerfile

```diff
@@ -1,6 +1,8 @@
 # Base build
 FROM rust:1.60-slim-bullseye AS build
 
+RUN apt-get update && apt-get install -y build-essential checkinstall zlib1g-dev
```

Comment:
The link complains about lack of pinning, you can add an ignore to this, no need to pin them.